Sources/SwiftSyntaxBuilderGeneration/main.swift

```diff
@@ -0,0 +1,193 @@
+
+          FunctionDecl(identifier: .identifier("`\(token.name.withFirstCharacterLowercased)`"),
+                       signature: signature,
+                       modifiersBuilder: {
+            if let text = token.text {
+              TokenSyntax.static.withLeadingTrivia(.newlines(1) + .docLineComment("/// The `\(text)` token"))
+            } else {
+              TokenSyntax.static
+            }
+          },
+                       bodyBuilder: {
+            FunctionCallExpr(calledExpression: MemberAccessExpr(base: "SyntaxFactory", name: "make\(token.name)"), leftParen: .leftParen, rightParen: .rightParen, trailingClosure: nil, argumentListBuilder: {
+              TupleExprElement(expression: IdentifierExpr("text"))
+            })
+
+            if token.requiresLeadingSpace {
+              createWithLeadingTriviaCall()
+            }
+
+            if token.requiresTrailingSpace {
+              createWithTrailingTriviaCall()
+            }
+          })
+        }
+      }
+      VariableDecl(letOrVarKeyword: .var,
+                   modifiersBuilder: { TokenSyntax.static.withLeadingTrivia(.newlines(1) + .docLineComment("/// The `eof` token") + .newlines(1)) },
+                   bindingsBuilder: {
+        // We need to use `CodeBlock` here to ensure there is braces around.
+        let body = CodeBlock {
+          FunctionCallExpr(calledExpression: MemberAccessExpr(base: "SyntaxFactory", name: "makeToken"),
+                           leftParen: .leftParen,
+                           rightParen: .rightParen,
+                           argumentListBuilder: {
+            TupleExprElement(expression: FunctionCallExpr(calledExpression: MemberAccessExpr(name: "eof")), trailingComma: .comma)
+            TupleExprElement(label: TokenSyntax.identifier("presence"), colon: .colon, expression: FunctionCallExpr(calledExpression: MemberAccessExpr(name: "present")))
+          })
+        }
+
+        createTokenSyntaxPatternBinding("eof", accessor: body)
+      })
+      VariableDecl(letOrVarKeyword: .var,
+                   modifiersBuilder: { TokenSyntax.static.withLeadingTrivia(.newlines(1) + .docLineComment("/// The `open` contextual token") + .newlines(1)) },
+                   bindingsBuilder: {
+        // We need to use `CodeBlock` here to ensure there is braces around.
+        let body = CodeBlock {
+          FunctionCallExpr(calledExpression: MemberAccessExpr(base: "SyntaxFactory", name: "makeContextualKeyword"),
+                           leftParen: .leftParen,
+                           rightParen: .rightParen,
+                           argumentListBuilder: {
+            TupleExprElement(expression: StringLiteralExpr("open"))
+          })
+
+          createWithTrailingTriviaCall()
+        }
+
+        createTokenSyntaxPatternBinding("open", accessor: body)
+      })
+    })
+}
+
+let tokenSyntax = tokensFile.buildSyntax(format: format)
+
+guard !CommandLine.arguments.isEmpty else {
```

Comment:
`CommandLine.arguments` is never empty, the first argument is always the executable that’s being called. I think this should be 
```suggestion
guard CommandLine.arguments.count > 1 else {
```